mm/2s2h/ShipUtils.cpp

```diff
@@ -29,6 +35,13 @@ std::unordered_map<s16, const char*> sceneNames = {
 #include "tables/scene_table.h"
 };
 
+std::vector<const char*> checkTrackerTextures = {
```

Comment:
Move this below the #undefs below and rename to like `miscTextures` with a comment explaining these aren't in the other few lists we iterate over.